osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -17,6 +17,7 @@ namespace osu.Game.Screens.Play
     /// Encapsulates gameplay timing logic and provides a <see cref="IGameplayClock"/> via DI for gameplay components to use.
     /// </summary>
     [Cached(typeof(IGameplayClock))]
+    [Cached(typeof(GameplayClockContainer))]
```

Comment:
Is there a reason this was added, or is some change missing from this PR?

Reply:
See https://github.com/ppy/osu/discussions/25713#discussion-5941592 - `HitErrorMeter` depends on being able to resolve by the `GameplayClockContainer` type specifically to subscribe to `OnSeek`.